### PR TITLE
display numbers#3

### PR DESCRIPTION
**Pull request by sarah-alsharif0**
display numbers on calculator when user clicks on them

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (Rahaf-96) [inline .vscode/settings.json]: delete `.vscode` folder please

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+}
```
Comment (someyaaltous) [inline .vscode/settings.json]: Delete this file

File: script.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        equals();
+    else if (target.classList.contains("clear")){
+        calculator.firstNum=null;
+        calculator.secondNum=null;
+        calculator.operator=null;
+        calcDisplay.value= 0;
+        }
+    else if (target.classList.contains("conv-btn"))
+        currencyConvert(target.innerText);
+    console.log(calculator);
```
Comment (Rahaf-96) [inline script.js]: remove `console.log` from here.

**Review by Rahaf-96 — APPROVED** [review]
Grat job Sarah! :star_struck: :heart:

**Comment by saharAdem** [comment]
Excellent Sara!

**Review by saharAdem — APPROVED** [review]

**Review by someyaaltous — APPROVED** [review]
Great job Sarah

**Pull request merged**